jerry-ext/debugger/debugger-tcp.c

```diff
@@ -167,7 +180,10 @@ jerryx_debugger_tcp_send (jerry_debugger_transport_header_t *header_p, /**< tcp
     }
 #endif /* __linux__ */
 
-    ssize_t sent_bytes = send (tcp_p->tcp_socket, message_p, message_length, 0);
+    jerryx_socket_ssize_t sent_bytes = send (tcp_p->tcp_socket,
+                                              (jerryx_socket_void_t *) message_p,
+                                              (jerryx_socket_size_t) message_length,
+                                              0);
```

Comment:
(minor style) indentation is off by one space (the last three lines should start where `tcp_p->...` starts on the first line)